tests/flytekit/unit/lazy_module/test_lazy_module.py

```diff
@@ -1,12 +1,13 @@
 import pytest
 
-from flytekit.lazy_import.lazy_module import LazyModule, lazy_module
+from flytekit.lazy_import.lazy_module import LazyModule, lazy_module, is_imported
 
 
 def test_lazy_module():
     mod = lazy_module("click")
     assert mod.__name__ == "click"
     mod = lazy_module("fake_module")
+    assert not is_imported("fake_module")
```

Comment:
This assertion passes even without `LAZY_MODULES.append(module.__name__)`.

Is there a test that fails on `master` and passes with this PR?

Reply:
updated test. it will fail on master